views/RgbLightView/RgbLightView.tsx

```diff
@@ -42,14 +46,16 @@ export const RgbLightView: FC<RgbLightViewProps> = ({ route }) => {
   return (
     <View style={styles.container}>
       <Navbar text={`${location} - ${nodeId} - ${rgbLight?.name}`} />
-      <View style={styles.content}>
-        <Typography
-          variant="body-medium"
-          text={`Aktualny kolor lampy to: ${rgbLight.values.V_RGB}`}
-        />
-        <Input text={color} onChange={onColorChange} />
-        <Button text="Zmień kolor" onPress={handleChangeColor} />
-      </View>
+      <ScrollView onScrollBeginDrag={() => Keyboard.dismiss()}>
```

Comment:
```suggestion
      <ScrollView onScrollBeginDrag={Keyboard.dismiss}>
```
Is it working? I didn't test it but seems fine